realm/realm-library/src/androidTest/java/io/realm/RealmAnnotationTests.java

```diff
@@ -227,6 +228,63 @@ public void primaryKey_isIndexed() {
         assertTrue(table.hasSearchIndex(table.getColumnIndex("id")));
     }
 
+    @Test
```

Comment:
Those three tests should be removed to `JNITableTest` since they cannot be compiled after OS and they are more like unit testing for `renamePrimaryKeyColumn`. And it doesn't really require the dependencies from defined schema. I would like to create the shemas with `Table`'s API, and test them in `JNITableTest`.